src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
create a global field Random and use it here, so you won't need to create its instance each time this method is called